gitlint/rules.py

```diff
@@ -243,14 +245,23 @@ def validate(self, commit):
             if first_line != "":
                 return [RuleViolation(self.id, "Second line is not empty", first_line, 2)]
 
-
 class BodyMinLength(CommitRule):
     name = "body-min-length"
     id = "B5"
-    options_spec = [IntOption('min-length', 20, "Minimum body length")]
+    options_spec = [IntOption('min-length', 20, "Minimum body length"),
+                    StrOption('ignore-body-checks-regex', '', 'Ignore body checks for tiles matching regex')]
+
 
     def validate(self, commit):
         min_length = self.options['min-length'].value
+        regex = self.options['ignore-body-checks-regex'].value
+        regex_valid = True if (self.options['ignore-body-checks-regex'].value) else False
+
+        if regex_valid and re.search(regex, commit.message.title):
+            logmsg = "Skipping %s (%s), because ignore-body-checks-regex matches." % (self.id, self.name)
+            LOG.info(logmsg)
```

Comment:
Do not use `%`, but pass the formatting string to the logger (because that is faster if the log message does not be printed):
```python
LOG.info("Skipping %s (%s), because ignore-body-checks-regex matches.", self.id, self.name)
```